SPECS/xs-opam-repo.spec

```diff
@@ -1,6 +1,6 @@
 %global package_speccommit e05d072ac22425dcfe80e765c548d04b3cf775ca
 %global usver 6.66.0
-%global xsver 1
+%global xsver 2
```

Comment:
We don't modify `xsver`. This belongs to XenServer packaging. 

Add `.1` in the Release tag definition, instead.

And in fact, there's already `.1`, so increment it to `.2`.